test/RegisterAccessNetworkTemplate.test.ts

```diff
@@ -0,0 +1,281 @@
+      secretId = await dleq.secretToPublic(secret)
+      encryptedPasswd = await dleq.encrypt(passwd, secret, providerPub)
+      cipher = dleq.bigToHex(encryptedPasswd)
+    })
+
+    it('should configure secret', async () => {
+      await accessCondition.addSecret(secretId, publisher)
+      const pid = await accessCondition.pointId(secretId)
+      await accessCondition.addPrice(pid, BigNumber.from(1), token.address, 20, publisher)
+    })
+
+    it('should register a DID', async () => {
+      await keeper.didRegistry.registerAttribute(didSeed, checksum, [], url, publisher.getId())
+    })
+
+    it('should generate the condition IDs', async () => {
+      conditionIdAccess = await accessCondition.generateIdWithSeed(
+        agreementId,
+        await accessCondition.hashValues(cipher, secretId, providerPub, buyerPub),
+      )
+      conditionIdLock = await lockPaymentCondition.generateIdWithSeed(
+        agreementId,
+        await lockPaymentCondition.hashValues(
+          did,
+          escrowPaymentCondition.getAddress(),
+          token.getAddress(),
+          amounts,
+          receivers,
+        ),
+      )
+      conditionIdEscrow = await escrowPaymentCondition.generateIdWithSeed(
+        agreementId,
+        await escrowPaymentCondition.hashValues(
+          did,
+          amounts,
+          receivers,
+          consumer.getId(),
+          escrowPaymentCondition.getAddress(),
+          token.getAddress(),
+          conditionIdLock[1],
+          conditionIdAccess[1],
+        ),
+      )
+      const i1 = await accessCondition.instance(agreementId, accessCondition.params(cipher, secretId, providerPub, buyerPub))
+      const i2 = await lockPaymentCondition.instance(agreementId, lockPaymentCondition.params(
+        did,
+        escrowPaymentCondition.getAddress(),
+        token.getAddress(),
+        amounts,
+        receivers,
+      ))
+      const i3 = await escrowPaymentCondition.instance(agreementId, escrowPaymentCondition.params(
+        did,
+        amounts,
+        receivers,
+        consumer.getId(),
+        escrowPaymentCondition.getAddress(),
+        token.getAddress(),
+        conditionIdLock[1],
+        conditionIdAccess[1],
+      ))
+      instances = [i1, i2, i3]
+    })
+
+    it('should have conditions types', async () => {
+      const conditionTypes = await accessTemplate.getConditionTypes()
+
+      assert.equal(conditionTypes.length, 3, 'Expected 3 conditions.')
+      assert.deepEqual(
+        [...conditionTypes].sort(),
+        [
+          accessCondition.getAddress(),
+          escrowPaymentCondition.getAddress(),
+          lockPaymentCondition.getAddress(),
+        ].sort(),
+        "The conditions doesn't match",
+      )
+    })
+
+    it('should have condition instances associated', async () => {
+      const conditionInstances = await accessTemplate.getConditions()
+
+      assert.equal(conditionInstances.length, 3, 'Expected 3 conditions.')
```

Comment:
this asset is duplicated with previous test L195

Reply:
Well there it was `getConditionTypes`